pyccel/stdlib/math/pyc_math_c.c

```diff
@@ -7,7 +7,7 @@
 #include <math.h>
 
 /*---------------------------------------------------------------------------*/
-int64_t        pyc_factorial(int64_t n)
+int64_t                     pyc_factorial(int64_t n)
```

Comment:
Is there any reason to add this giant spacing?

Reply:
I just wanted all the functions to be inlined vertically, to make the code readable after adding the `inline` keyword to some functions.